eve/main.yml

```
@@ -125,6 +126,29 @@ stages:
         urls:
           - "*.iso"
 
+  build-base-iso:
+    worker:
+      type: kube_pod
+      path: eve/workers/build-iso/pod.yaml
+      images:
+        build-iso: eve/workers/build-iso
+    steps:
+    - Git: *git_pull
+    - ShellCommand: *wait_docker_daemon
+    - ShellCommand: *private_registry_login
+    - ShellCommand:
+        name: 'Build ISO'
+        haltOnFailure: true
+        workdir: build/solution-base
+        command: ./build.sh
+        env:
+          DOCKER_SOCKET: 'http://localhost:2375'
+    - Upload:
+        name: Upload ISO
+        source: 'solution/_build'
+        urls:
+          - "*.iso"
```

same yaml trick here:

line 123
```yaml
    - Upload: *upload-iso
```

```suggestion
    - Upload: *upload-iso
```

---

the source is different in each ISO so not sure if this works

---

woops, true